webapp/app/js/models/resultsets.js

```diff
@@ -61,30 +62,52 @@ treeherder.factory('ThResultSetModel', [
 
     var registerResultSetPollers = function(){
 
-        if( doResultSetPolling() ){
-            // Register resultset poller if it's not registered
-            $interval(function(){
+        // these params will be passed in each time we poll to remain
+        // within the constraints of the URL params
+        var rsPollingParams = _.pick($location.search(), rsPollingKeys);
 
-                if( (repositories[$rootScope.repoName].resultSets.length > 0) &&
-                    (repositories[$rootScope.repoName].loadingStatus.prepending === false) ){
+        // Register resultset poller if it's not registered
+        $interval(function(){
 
+            // This case is if we already have at least 1 resultset and we're
+            // polling for more.
+            // If we already have one, and the revision param is passed in,
+            // don't poll for more, because "there can be only one."
+            if( (repositories[$rootScope.repoName].resultSets.length > 0) &&
```

Comment:
Was just curious about the non-space after the `if` and the white space between the `( ( `, similarly on the `else if(` on line 88. For my own interest mostly :) ...is it a particular editor convention on long conditions?